src/components/element-editor/validation/validation.tsx

```diff
@@ -19,7 +19,7 @@ export const Validation = () => {
         name: "validation",
         shouldUnregister: true,
     });
-    const schema = SchemaHelper.buildSchema(EElementType.EMAIL);
+    const schema = SchemaHelper.buildSchema(EElementType.NUMERIC);
```

Comment:
move `elementType` above `schema` and apply it to `buildSchema()`
```suggestion
    const elementType = watch("type", focusedElement.element.type);
    const schema = SchemaHelper.buildSchema(elementType);
```